Build failure: flyctl-0.2.69

### Steps To Reproduce

Steps to reproduce the behavior:
1. build flyctl-0.2.69

### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/a1jav5bfxgn5h3pny396ymw7wpday7w8-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
applying patch /nix/store/xxrf1rf1c7977d7ab6w4dzi3ijjb6n8l-disable-auto-update.patch
patching file internal/config/config.go
Hunk #1 succeeded at 159 with fuzz 1 (offset 18 lines).
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Building subPackage ./.
buildPhase completed in 30 seconds
@nix { "action": "setPhase", "phase": "checkPhase" }
Running phase: checkPhase
?   	github.com/superfly/flyctl	[no test files]
?   	github.com/superfly/flyctl/agent	[no test files]
?   	github.com/superfly/flyctl/agent/internal/proto	[no test files]
?   	github.com/superfly/flyctl/agent/server	[no test files]
?   	github.com/superfly/flyctl/cmd/audit	[no test files]
?   	github.com/superfly/flyctl/doc	[no test files]
?   	github.com/superfly/flyctl/flyctl	[no test files]
?   	github.com/superfly/flyctl/flypg	[no test files]
?   	github.com/superfly/flyctl/gql	[no test files]
ok  	github.com/superfly/flyctl/helpers	0.017s
ok  	github.com/superfly/flyctl/internal/appconfig	0.054s
?   	github.com/superfly/flyctl/internal/build/imgsrc/builtins	[no test files]
?   	github.com/superfly/flyctl/internal/buildinfo	[no test files]
?   	github.com/superfly/flyctl/internal/cli	[no test files]
ok  	github.com/superfly/flyctl/internal/build/imgsrc	0.059s
ok  	github.com/superfly/flyctl/internal/cache	0.016s
?   	github.com/superfly/flyctl/internal/cmdfmt	[no test files]
?   	github.com/superfly/flyctl/internal/cmdutil	[no test files]
?   	github.com/superfly/flyctl/internal/cmdutil/preparers	[no test files]
ok  	github.com/superfly/flyctl/internal/cli_test	0.060s
?   	github.com/superfly/flyctl/internal/command/agent	[no test files]
?   	github.com/superfly/flyctl/internal/command/apps	[no test files]
?   	github.com/superfly/flyctl/internal/command/auth	[no test files]
?   	github.com/superfly/flyctl/internal/command/auth/webauth	[no test files]
?   	github.com/superfly/flyctl/internal/command/certificates	[no test files]
?   	github.com/superfly/flyctl/internal/command/checks	[no test files]
?   	github.com/superfly/flyctl/internal/command/config	[no test files]
?   	github.com/superfly/flyctl/internal/command/console	[no test files]
?   	github.com/superfly/flyctl/internal/command/consul	[no test files]
?   	github.com/superfly/flyctl/internal/command/create	[no test files]
?   	github.com/superfly/flyctl/internal/command/curl	[no test files]
?   	github.com/superfly/flyctl/internal/command/dashboard	[no test files]
ok  	github.com/superfly/flyctl/internal/command	0.052s
?   	github.com/superfly/flyctl/internal/command/destroy	[no test files]
?   	github.com/superfly/flyctl/internal/command/dig	[no test files]
?   	github.com/superfly/flyctl/internal/command/dnsrecords	[no test files]
?   	github.com/superfly/flyctl/internal/command/docs	[no test files]
?   	github.com/superfly/flyctl/internal/command/doctor	[no test files]
?   	github.com/superfly/flyctl/internal/command/doctor/diag	[no test files]
?   	github.com/superfly/flyctl/internal/command/domains	[no test files]
?   	github.com/superfly/flyctl/internal/command/errors	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/arcjet	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/core	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/enveloop	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/kafka	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/kubernetes	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/sentry	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/supabase	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/tigris	[no test files]
?   	github.com/superfly/flyctl/internal/command/extensions/vector	[no test files]
?   	github.com/superfly/flyctl/internal/command/history	[no test files]
?   	github.com/superfly/flyctl/internal/command/image	[no test files]
?   	github.com/superfly/flyctl/internal/command/incidents	[no test files]
?   	github.com/superfly/flyctl/internal/command/incidents/hosts	[no test files]
?   	github.com/superfly/flyctl/internal/command/info	[no test files]
?   	github.com/superfly/flyctl/internal/command/ips	[no test files]
?   	github.com/superfly/flyctl/internal/command/jobs	[no test files]
?   	github.com/superfly/flyctl/internal/command/launch	[no test files]
?   	github.com/superfly/flyctl/internal/command/launch/plan	[no test files]
?   	github.com/superfly/flyctl/internal/command/lfsc	[no test files]
?   	github.com/superfly/flyctl/internal/command/logs	[no test files]
?   	github.com/superfly/flyctl/internal/command/machine	[no test files]
?   	github.com/superfly/flyctl/internal/command/metrics	[no test files]
?   	github.com/superfly/flyctl/internal/command/move	[no test files]
?   	github.com/superfly/flyctl/internal/command/open	[no test files]
?   	github.com/superfly/flyctl/internal/command/orgs	[no test files]
?   	github.com/superfly/flyctl/internal/command/ping	[no test files]
?   	github.com/superfly/flyctl/internal/command/platform	[no test files]
[33mWARN[0m Updating the mount path for volume vol_attached on machine ab1234567890 from /update-me to /data due to fly.toml [mounts] destination value

[33mWARN[0m Machine ab1234567890 has volume 'replace-me' attached but fly.toml have a different name: 'data'

[33mWARN[0m Machine ab1234567890 has a volume attached but fly.toml doesn't have a [mounts] section

[33mWARN[0m Updating the mount path for volume vol_attached on machine ab1234567890 from /update-me to /data due to fly.toml [mounts] destination value

[33mWARN[0m Machine ab1234567890 has volume 'replace-me' attached but fly.toml have a different name: 'data'

[33mWARN[0m Machine ab1234567890 has a volume attached but fly.toml doesn't have a [mounts] section

--- FAIL: TestCommand_Execute (0.30s)
panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map

goroutine 71 [running]:
testing.tRunner.func1.2({0x103fb0b20, 0x104277e90})
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/testing/testing.go:1631 +0x1c4
testing.tRunner.func1()
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/testing/testing.go:1634 +0x33c
panic({0x103fb0b20?, 0x104277e90?})
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/runtime/panic.go:770 +0x124
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End.deferwrap1()
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:381 +0x2c
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End(0x1400014c480, {0x0, 0x0, 0x1400034ba78?})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:419 +0x8ac
panic({0x103fb0b20?, 0x104277e90?})
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/runtime/panic.go:770 +0x124
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End.deferwrap1()
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:381 +0x2c
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End(0x1400014cf00, {0x0, 0x0, 0x1400034b350?})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:419 +0x8ac
panic({0x103fb0b20?, 0x104277e90?})
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/runtime/panic.go:770 +0x124
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End.deferwrap1()
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:381 +0x2c
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End(0x1400014d200, {0x0, 0x0, 0x14000584380?})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:419 +0x8ac
panic({0x103fb0b20?, 0x104277e90?})
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/runtime/panic.go:770 +0x124
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End.deferwrap1()
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:381 +0x2c
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End(0x1400014d380, {0x0, 0x0, 0x14000584380?})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/go.opentelemetry.io/otel/sdk/trace/span.go:419 +0x8ac
panic({0x103fb0b20?, 0x104277e90?})
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/runtime/panic.go:770 +0x124
github.com/getsentry/sentry-go/internal/ratelimit.Map.Merge(...)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/internal/ratelimit/map.go:43
github.com/getsentry/sentry-go.(*HTTPSyncTransport).SendEventWithContext(0x14000793880, {0x10429a4f0, 0x1052cb8e0}, 0x1400064c588)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/transport.go:693 +0x4c8
github.com/getsentry/sentry-go.(*HTTPSyncTransport).SendEvent(0x140002490e0?, 0x1400064c588?)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/transport.go:644 +0x30
github.com/getsentry/sentry-go.(*Client).processEvent(0x140002490e0, 0x1400064c588?, 0x1400015c180, {0x104280300, 0x1400013c5a0})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/client.go:637 +0x30c
github.com/getsentry/sentry-go.(*Client).CaptureEvent(...)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/client.go:447
github.com/getsentry/sentry-go.(*Client).CaptureException(0x140002490e0, {0x10427d5b8?, 0x14000278940?}, 0x1400015c180, {0x104280300, 0x1400013c5a0})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/client.go:428 +0x58
github.com/getsentry/sentry-go.(*Hub).CaptureException(0x140006a79e0, {0x10427d5b8, 0x14000278940})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/hub.go:260 +0x98
github.com/getsentry/sentry-go.CaptureException(...)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/sentry.go:45
github.com/superfly/flyctl/internal/sentry.CaptureException.func1(0x1400013c5a0)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/sentry/sentry.go:92 +0x88
github.com/getsentry/sentry-go.(*Hub).WithScope(0x140006a79e0, 0x1400034a3c8)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/hub.go:200 +0x60
github.com/getsentry/sentry-go.WithScope(...)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/getsentry/sentry-go/sentry.go:91
github.com/superfly/flyctl/internal/sentry.CaptureException({0x10427d5b8?, 0x14000278940?}, {0x1400034a8e0?, 0x7?, 0x0?})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/sentry/sentry.go:87 +0xd0
github.com/superfly/flyctl/internal/build/imgsrc.(*Resolver).createBuildGql(0x140003ba1c0, {0x10429a848, 0x1400013a480}, {0x140003ba200, 0x2, 0x2}, 0x1400034a9b8)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/build/imgsrc/resolver.go:333 +0x570
github.com/superfly/flyctl/internal/build/imgsrc.(*Resolver).createImageBuild(0x140003ba1c0, {0x10429a848, 0x1400013a480}, {0x1400034abb8?, 0x0?, 0x0?}, {{0x14000350d50, 0xa}, {0x14000054aa0, 0x4c}, ...})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/build/imgsrc/resolver.go:285 +0x180
github.com/superfly/flyctl/internal/build/imgsrc.(*Resolver).ResolveReference(0x140003ba1c0, {0x10429a848, 0x1400013a450}, 0x140003b41e0, {{0x14000350d50, 0xa}, {0x14000054aa0, 0x4c}, {0x103a59d5f, 0x43}, ...})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/build/imgsrc/resolver.go:170 +0x19c
github.com/superfly/flyctl/internal/command/deploy.determineImage({0x10429a848, 0x1400013a150}, 0x140002bd680, 0x1)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/command/deploy/deploy_build.go:89 +0x1224
github.com/superfly/flyctl/internal/command/deploy.DeployWithConfig({0x10429a848, 0x1400013a150}, 0x140002bd680, 0x3?)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/command/deploy/deploy.go:303 +0x200
github.com/superfly/flyctl/internal/command/deploy.(*Command).run(0x1052cb8e0?, {0x10429a848, 0x14000333a40})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/command/deploy/deploy.go:277 +0xc18
github.com/superfly/flyctl/internal/command/deploy.New.New.newRunE.func2(0x140002ea008, {0x1400012ab20?, 0x4?, 0x1039831a6?})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/command/command.go:140 +0x188
github.com/spf13/cobra.(*Command).execute(0x140002ea008, {0x1400012aac0, 0x2, 0x2})
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/spf13/cobra/command.go:983 +0x840
github.com/spf13/cobra.(*Command).ExecuteC(0x140002ea008)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/spf13/cobra/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/spf13/cobra/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/vendor/github.com/spf13/cobra/command.go:1032
github.com/superfly/flyctl/internal/command/deploy_test.TestCommand_Execute(0x14000694d00)
	/private/tmp/nix-build-flyctl-0.2.69.drv-0/source/internal/command/deploy/deploy_test.go:60 +0x6a8
testing.tRunner(0x14000694d00, 0x104272290)
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/testing/testing.go:1689 +0xec
created by testing.(*T).Run in goroutine 1
	/nix/store/m4ip3yyqg4k2pilbyfczgiy8xqvh4554-go-1.22.3/share/go/src/testing/testing.go:1742 +0x318
FAIL	github.com/superfly/flyctl/internal/command/deploy	0.371s
?   	github.com/superfly/flyctl/internal/command/proxy	[no test files]
?   	github.com/superfly/flyctl/internal/command/redis	[no test files]
?   	github.com/superfly/flyctl/internal/command/regions	[no test files]
?   	github.com/superfly/flyctl/internal/command/releases	[no test files]
?   	github.com/superfly/flyctl/internal/command/restart	[no test files]
?   	github.com/superfly/flyctl/internal/command/resume	[no test files]
?   	github.com/superfly/flyctl/internal/command/root	[no test files]
ok  	github.com/superfly/flyctl/internal/command/postgres	0.066s
ok  	github.com/superfly/flyctl/internal/command/scale	0.094s
?   	github.com/superfly/flyctl/internal/command/services	[no test files]
?   	github.com/superfly/flyctl/internal/command/settings	[no test files]
?   	github.com/superfly/flyctl/internal/command/ssh	[no test files]
ok  	github.com/superfly/flyctl/internal/command/secrets	0.054s
?   	github.com/superfly/flyctl/internal/command/storage	[no test files]
?   	github.com/superfly/flyctl/internal/command/suspend	[no test files]
?   	github.com/superfly/flyctl/internal/command/tokens	[no test files]
?   	github.com/superfly/flyctl/internal/command/version	[no test files]
?   	github.com/superfly/flyctl/internal/command/volumes	[no test files]
?   	github.com/superfly/flyctl/internal/command/volumes/lsvd	[no test files]
?   	github.com/superfly/flyctl/internal/command/volumes/snapshots	[no test files]
?   	github.com/superfly/flyctl/internal/command/wireguard	[no test files]
?   	github.com/superfly/flyctl/internal/command_context	[no test files]
ok  	github.com/superfly/flyctl/internal/command/status	0.054s
?   	github.com/superfly/flyctl/internal/ctrlc	[no test files]
?   	github.com/superfly/flyctl/internal/env	[no test files]
?   	github.com/superfly/flyctl/internal/filemu	[no test files]
?   	github.com/superfly/flyctl/internal/flag	[no test files]
?   	github.com/superfly/flyctl/internal/flag/completion	[no test files]
?   	github.com/superfly/flyctl/internal/flag/flagctx	[no test files]
?   	github.com/superfly/flyctl/internal/flag/flagnames	[no test files]
?   	github.com/superfly/flyctl/internal/flapsutil	[no test files]
?   	github.com/superfly/flyctl/internal/flyerr	[no test files]
?   	github.com/superfly/flyctl/internal/flyutil	[no test files]
?   	github.com/superfly/flyctl/internal/format	[no test files]
?   	github.com/superfly/flyctl/internal/future	[no test files]
ok  	github.com/superfly/flyctl/internal/config	0.018s
?   	github.com/superfly/flyctl/internal/heroku	[no test files]
?   	github.com/superfly/flyctl/internal/httptracing	[no test files]
?   	github.com/superfly/flyctl/internal/incidents	[no test files]
?   	github.com/superfly/flyctl/internal/inmem	[no test files]
?   	github.com/superfly/flyctl/internal/instrument	[no test files]
ok  	github.com/superfly/flyctl/internal/haikunator	0.013s
?   	github.com/superfly/flyctl/internal/logger/logfile	[no test files]
ok  	github.com/superfly/flyctl/internal/logger	0.014s
?   	github.com/superfly/flyctl/internal/metrics	[no test files]
?   	github.com/superfly/flyctl/internal/mock	[no test files]
ok  	github.com/superfly/flyctl/internal/machine	0.023s
?   	github.com/superfly/flyctl/internal/release	[no test files]
?   	github.com/superfly/flyctl/internal/render	[no test files]
?   	github.com/superfly/flyctl/internal/sentry	[no test files]
?   	github.com/superfly/flyctl/internal/spinner	[no test files]
?   	github.com/superfly/flyctl/internal/sort	[no test files]
ok  	github.com/superfly/flyctl/internal/prompt	0.017s
ok  	github.com/superfly/flyctl/internal/set	0.014s
?   	github.com/superfly/flyctl/internal/statuslogger	[no test files]
?   	github.com/superfly/flyctl/internal/tracing	[no test files]
?   	github.com/superfly/flyctl/internal/update	[no test files]
?   	github.com/superfly/flyctl/internal/watch	[no test files]
?   	github.com/superfly/flyctl/internal/wireguard	[no test files]
?   	github.com/superfly/flyctl/ip	[no test files]
?   	github.com/superfly/flyctl/logs	[no test files]
?   	github.com/superfly/flyctl/proxy	[no test files]
ok  	github.com/superfly/flyctl/internal/state	0.018s
ok  	github.com/superfly/flyctl/internal/task	0.021s
ok  	github.com/superfly/flyctl/internal/version	0.016s
ok  	github.com/superfly/flyctl/iostreams	0.016s
ok  	github.com/superfly/flyctl/retry	0.019s
?   	github.com/superfly/flyctl/scripts/clean-up-preflight-apps	[no test files]
?   	github.com/superfly/flyctl/ssh	[no test files]
?   	github.com/superfly/flyctl/terminal	[no test files]
?   	github.com/superfly/flyctl/tools/distribute	[no test files]
?   	github.com/superfly/flyctl/tools/distribute/bundle	[no test files]
?   	github.com/superfly/flyctl/tools/distribute/flypkgs	[no test files]
?   	github.com/superfly/flyctl/tools/version	[no test files]
?   	github.com/superfly/flyctl/wg	[no test files]
ok  	github.com/superfly/flyctl/scanner	0.021s
ok  	github.com/superfly/flyctl/tools/version/relmeta	0.013s
FAIL

```

### Additional context

Looks like recent bump https://github.com/NixOS/nixpkgs/pull/319808 with failing ofborg was merged and attempted to be backported right away... https://logs.ofborg.org/?attempt_id=a84fd283-c3c3-40f6-b90e-9229b7dea17a&key=nixos%2Fnixpkgs.319808

### Notify maintainers
@adtya @jsierles @techknowlogick @RaghavSood @teutat3s
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
- system: `"aarch64-darwin"`
 - host os: `Darwin 23.5.0, macOS 14.5`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.22.1`
 - nixpkgs: `/nix/store/qqwr649pc0qprc9lw2fmdsi1km6p7q2h-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/web/flyctl/default.nix
+++ b/pkgs/development/web/flyctl/default.nix
@@ -35,12 +35,14 @@ buildGoModule rec {
   '';
 
   # We override checkPhase to be able to test ./... while using subPackages
+  # Temporary fix for https://github.com/superfly/flyctl/issues/3642. We
+  # should go back to buildGoDir instead of go test once that is resolved.
   checkPhase = ''
     runHook preCheck
     # We do not set trimpath for tests, in case they reference test assets
     export GOFLAGS=''${GOFLAGS//-trimpath/}
 
-    buildGoDir test ./...
+    go test ./...
 
     runHook postCheck
   '';

--- a/pkgs/development/web/flyctl/default.nix
+++ b/pkgs/development/web/flyctl/default.nix
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "flyctl";
-  version = "0.2.69";
+  version = "0.2.71";
 
   src = fetchFromGitHub {
     owner = "superfly";
     repo = "flyctl";
     rev = "v${version}";
-    hash = "sha256-6SPJt+az+C0amCnRScp85JqUfnWswHdS1OtOMa40Haw=";
+    hash = "sha256-d2qoTRWuUh7Kn0uqT3fIlB8BbFJmyVEIlfInS3m+etc=";
   };
 
-  vendorHash = "sha256-O5lUw0XzQUoy3mAmSpeW6WfBgg0FLxXkk7hCUAS2+o8=";
+  vendorHash = "sha256-iRZrjkWQxuUW/YM5TygFt+g8suM5iLGsWsCt4QQOX3M=";
 
   subPackages = [ "." ];
 

